.script/tests/KqlvalidationsTests/CustomTables/Ubiquiti.json

```diff
@@ -0,0 +1,153 @@
+{
+    "Name": "UbiquitiAuditEvent",
```

Comment:
If we change it to Ubiquiti_CL, Kql validation tests will fail because detection rules are based on parser UbiquitiAuditEvent.
Do we need to add new mapping for Ubiquiti_CL instead of changing UbiquitiAuditEvent?

Reply:
@shainw : Can you please tell us if we need to add new mapping for Ubiquiti_CL or not?

Reply:
@shainw : Can you please help us here?

Reply:
It should be the Parser name if that is what the Solution will include.